### ISSUE TITLE
Invalid integer: quipCount

### ISSUE THREAD

**Issue by thelifeandtimes** [tloncorp/landscape-apps#2677]
```
Error: Invalid integer: quipCount
    at Cu (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/index.ad30a98e.js:1:24087)
    at ce (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/index.ad30a98e.js:1:24488)
    at u (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/index.ad30a98e.js:1:3435)
    at https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/index.ad30a98e.js:36:81758
    at Array.map (<anonymous>)
    at zh (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/index.ad30a98e.js:36:81739)
    at $7 (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/index.ad30a98e.js:48:229163)
    at Kl (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884)
    at mf (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:8723)
    at pf (https://sarlev-sarsen.rooftopdao.io/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:947)
```

tried to add a notebook post in a newly created notebook hosted by ~minder-folden. Was able to successfully `|hi ~minder-folden`. Notebook post `save` would give a perpetual spinner. shut down, melded, rebooted and then in trying to access the notebook I got this error.

**Comment by jamesacklin** [comment]
@thelifeandtimes Can you try again? This was fixed in the latest release.

**Comment by thelifeandtimes** [comment]
TL;DR: It's working as of two or three days ago (before the big "pals in groups update")

Longer Update: I have been able to make updates to this notebook post as of ~3 days ago. There was some trouble where joining a newly created channel (group host being different from channel host) where I could not see any messages but I could successfully post messages to the chat. Similarly, apparently the notebook post I was trying to put up was actually visible to others (even though I was stuck on the saving spinner). I tried leaving the channel and rejoining with no success; left the group and was reinvited, again with no success, and then re-left the channel and rejoined the channel and that seemed to work it out. Probably can consider this specific issue closed but if it is something you are seeing replicated elsewhere give a shout and I'm happy to further explore with you guys.

**Comment by patosullivan** [comment]
I suspect his cache wasn't properly busted for some reason.

**Comment by thelifeandtimes** [comment]
I did do hard refreshes, but don't think I rebooted Brave. I also have been using the Groups and Talk PWAs via brave and think there are some odd behaviors that happen between the two as well. I.e. the recent data collection notification read/unread state is stored via cookies which are separate between the originating browser and the PWA, so I experienced it twice: once in the groups PWA, and againafter opening a groups link from the talk PWA which opened in the originating browser.
